exporter/exporter.go

```diff
@@ -57,8 +59,106 @@ resource "lightstep_metric_dashboard" "exported_dashboard" {
 }
 `
 
-func escapeSpanQuery(input string) string {
-	return strings.Replace(input, "\"", "\\\"", -1)
+const unifiedDashboardTemplate = `
+resource "lightstep_dashboard" "exported_dashboard" {
+  project_name = var.project
+  dashboard_name = "{{.Attributes.Name}}"
+{{range .Attributes.Charts}}
+  chart {
+    name = "{{.Title}}"
+    rank = "{{.Rank}}"
+    type = "{{.ChartType}}"
+{{range .MetricQueries}}
+    query {
+      query_name          = "{{.Name}}"
+      display             = "{{.Display}}"
+      hidden              = {{.Hidden}}
+      query_string        = {{escapeQueryString .TQLQuery}}
+	}
+{{end}}
+  }
+{{end}}
+}
+`
+
+func escapeHCLString(input string) string {
```

Comment:
Replaces `escapeSpanQuery` which was only escaping double-quote.  There weren't any bug logged on this AFAIK, but the change seems correct.